frontend/packages/console-app/src/components/guided-tours/GuidedTourItem.scss

```diff
@@ -0,0 +1,3 @@
+.odc-guided-tour-item {
+  height: 100%;
+}
```

Comment:
Remove the line clamp behavior until https://github.com/patternfly/patternfly-react/issues/4451 is fixed.
```
& .catalog-tile-pf-description .has-footer {
    display: block;
    -webkit-line-clamp: unset;
}
```

Reply:
added